src/core/border_router/routing_manager.hpp

```diff
@@ -1275,7 +1288,12 @@ class RoutingManager : public InstanceLocator
 #endif
 
 #if OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
-    PdPrefixManager mPdPrefixManager;
+    PdPrefixManager   mPdPrefixManager;
+    bool              mRequirePD;
+    int64_t           mNumPlatformRaReceived;
+    int64_t           mLastPlatformRaTimestamp;
+    otPacketsAndBytes mInboundInternet;
+    otPacketsAndBytes mOutboundInternet;
```

Comment:
Suggest moving all these variable to `PdPrefixManager` class.

`PdPrefixManager` class is responsible for managing PdPrefix. It should itself keep track of all of this info.

Reply:
I prefer to keep them in ip6.cpp to handle all the border routing counters together

Reply:
My suggestion is about `mNumPlatformRaReceived` and `mLastPlatformRaTimestamp` variables.